jax/lax_linalg.py

```diff
@@ -326,8 +320,6 @@ def triangular_solve_shape_rule(a, b, left_side=False, **unused_kwargs):
 
 def triangular_solve_jvp_rule_a(
     g_a, ans, a, b, left_side, lower, transpose_a, conjugate_a, unit_diagonal):
-  if g_a is ad_util.zero:
```

Comment:
Even if the AD interpreter does not invoke the jvp rule if all inputs are symbolic zeros, this function has two inputs, and it is possible that only one is a symbolic zero.

Reply:
Actually, I take it back. This primitive uses `defjvp2` which handles that case.